### PR TITLE
Add support for swift package

### PR DESCRIPTION
**Pull request by wzio**
It's a great to see an updated version of this library. I added support for swift packages

### CODE REVIEW

File: Source/ResourceLoaderDelegate.swift [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 import Foundation
 import AVFoundation
+import UIKit
```
Comment (sukov) [inline Source/ResourceLoaderDelegate.swift:10]:
```suggestion
```

**Review by sukov — COMMENTED** [review]
Thanks for the contribution. I requested only one change, and after that I will merge it.

**Comment by wzio** [comment]
`ResourceLoaderDelegate.swift` references `UIApplication` at line 39
```swift
NotificationCenter.default.addObserver(self, selector: #selector(handleAppWillTerminate), name: UIApplication.willTerminateNotification, object: nil)
```
So you have to import UIKit
CocoaPods [imports UIKit by default](https://github.com/CocoaPods/CocoaPods/pull/6816) for all iOS-targeted libraries, so it won't compile error here. Whereas the swift package doesn't have this default behaviour.

**Comment by sukov** [comment]
> `ResourceLoaderDelegate.swift` references `UIApplication` at line 39
> 
> ```swift
> NotificationCenter.default.addObserver(self, selector: #selector(handleAppWillTerminate), name: UIApplication.willTerminateNotification, object: nil)
> ```
> 
> So you have to import UIKit CocoaPods [imports UIKit by default](https://github.com/CocoaPods/CocoaPods/pull/6816) for all iOS-targeted libraries, so it won't compile error here. Whereas the swift package doesn't have this default behaviour.

Oh, I see. Thanks for the clarification!

**Review by sukov — APPROVED** [review]

**Pull request merged**